lib/libsys/intro.2

```diff
@@ -68,12 +68,14 @@ variable that is compatible with the previous definition.
 .Pp
 When a system call detects an error,
 it returns an integer value
-indicating failure (usually -1)
+indicating failure
+.Pq usually -1
 and sets the variable
 .Va errno
 accordingly.
-(This allows interpretation of the failure on receiving
-a -1 and to take action accordingly.)
+.Po This allows interpretation of the failure on receiving
+-1 and to take action accordingly
+.Pc
 Successful calls never set
 .Va errno ;
 once set, it remains until another error occurs.
```

Comment:
_No changes required, just a comment._

Having spent far too much time in the implementation, I find the phrasing in this paragraph a bit odd. From my perspective the system call is the thing in the kernel beyond the machine-dependent trap invocation. The invocation, detection that the system call returned an error and storing that in `errno` is a property of the stub. This probably isn't a useful distinction for users today, but there is some interest in an errno-free interface to syscalls were they return 0 or an error (and things that return other things today grow an argument for that purpose).